samples/bluetooth/central_uart/src/main.c

```diff
@@ -389,7 +389,7 @@ static void connected(struct bt_conn *conn, uint8_t conn_err)
 	}
 
 	err = bt_scan_stop();
-	if ((!err) && (err != -EALREADY)) {
+	if (err) {
```

Comment:
Thank you, @vidarbe, for improving this :) I no longer maintain this sample but we may want to desist from printing the error log if the error code is equal to `-EALREADY`. That was probably my intention when I wrote this code years ago:

```suggestion
	if (err && (err != -EALREADY)) {
```

Reply:
I assumed that was your intention too, but I did some quick testing before creating the PR and didn’t see the -EALREADY error, which is why I thought it was fine to remove it. I’m not completely opposed to changing this, but I feel it’s better not to mask the error. The only side effect in this case would be an error message in the log.